includes/utils/class-amp-css-utils.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class AMP_CSS_Utils
+{
+	public static function extract_and_minify_css_file($cssFile)
+	{
+		// Load CSS file contents
+		$minified_css = file_get_contents($cssFile);
+		// Remove comments
+		$minified_css = preg_replace('!/\*[^*]*\*+([^/][^*]*\*+)*/!', '', $minified_css);
+		// Remove space after colons
+		$minified_css = str_replace(': ', ':', $minified_css);
```

Comment:
Instead of lines `12` and `14` why not just this? 

    $minified_css = preg_replace( '#\s+#ms', '', $minified_css);`